.github/workflows/ReleaseStaticBinaries.yml

```diff
@@ -16,32 +16,19 @@ jobs:
         include:
           - os: ubuntu-latest
             target: x86_64-unknown-linux-musl
+          - os: ubuntu-latest
+            target: aarch64-unknown-linux-musl
           - os: macos-latest
             target: x86_64-apple-darwin
           - os: macos-latest
             target: aarch64-apple-darwin
+          - os: windows-latest
+            target: x86_64-pc-windows-msvc
 
     steps:
-    - uses: actions/checkout@v3
-
-    - name: Install Rust
-      uses: dtolnay/rust-toolchain@master
-      with:
-        toolchain: stable
-        target: ${{ matrix.target }}
-
-    - name: Install cargo-rs
-      shell: bash
-      run: |
-        cargo install cross
-
-    - name: Build
-      run: cross build --target ${{ matrix.target }} --release
-
-    - name: Upload binaries to GitHub Release
-      uses: svenstaro/upload-release-action@v2
-      with:
-        repo_token: ${{ secrets.GITHUB_TOKEN }}
-        file: target/${{ matrix.target }}/release/tspin
-        asset_name: tspin-${{ matrix.target}}
-        tag: ${{ github.ref }}
+      - uses: actions/checkout@v3
+      - uses: taiki-e/upload-rust-binary-action@v1
```

Comment:
I didn't know this action. Thanks @supleed2!

The `svenstaro/upload-release-action` action is [most commonly](https://grep.app/search?q=svenstaro/upload-release-action). But I don't think this is a good argument against this action :sweat_smile: 

Approved!

Reply:
Oh cool, before this I had never seen the `svenstaro` one before hah! Thanks for being so chill.